.github/workflows/verify.yaml

```diff
@@ -35,9 +35,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
       - name: Build All Variants
-        run: mvn clean install -Pprod -Pjpa -Pinfinispan -Pkafka -Pstreams -Pasyncmem -DskipTests
-      - name: Verify (Standard)
-        run: mvn test package -Pprod -Pjpa -Pasyncmem
```

Comment:
This section runs the unit tests - I don't see this added to the other job, so I think it should stay.

Reply:
ok

Reply:
unit tests are executed here https://github.com/Apicurio/apicurio-registry/pull/681/files#diff-93216a1157045809d672cb06668f4bb2R38

Reply:
Ah yes - you've removed the `-DskipTests` for that.  Unfortunately I think that will fail for the **streams** and **kafka** profiles still...